services/QuillLMS/spec/services/quill_big_query/materlialized_view_refresher_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe QuillBigQuery::MaterializedViewRefresher do
+  let(:query_key) { 'reporting_sessions_view' }
+
+  describe 'config' do
+    subject {described_class.new(query_key)}
+
+    it 'should find SQL file' do
+      expect { subject.send(:create_sql)}.not_to raise_error
+    end
+  end
+
+  describe 'run' do
+    let(:report_sessions_sql) {'SELECT 2'}
```

Comment:
Is this line needed?

Reply:
Ah good catch, my intention was to use this for documentation that that is coming from the SQL file, which might be less obvious when this is not in PR, but I left it out. Let me put it back in.